docs/guides/hooks.md

```diff
@@ -99,6 +103,53 @@ videojs.hook('setup', function(player) {
 videojs('some-id', {autoplay: true, controls: true});
 ```
 
+### beforeerror
+
+`beforeerror` occurs just as we get an error on the player. This allows plugins or other custom code to intercept the error and modify it to be something else.
+`error` is can be an object such as `{code: 4}` or something with a message. It potentially includes a `cause` property too. Alternatively, it could be `null` which means that the current error will be getting cleared.
```

Comment:
```suggestion
`error` is an object that has a `code` and `message` property and potentially a `cause` property too. Alternatively, it could be `null` which means that the current error should be cleared.
```

Reply:
Looks like error can be a string or just a number too, so, I'll update this and the code based on expectations.